meta-balena-common/recipes-support/ca-certificates/ca-certificates/extract-balena-ca.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Wondering whether this unit should run whenever the config.json custom CA is updated.
I guess the supervisor will reboot the unit after modifying config.json anyway, so it's good as is. But conceptually I don't know why the reboot is needed in all cases when changes could be applied on the fly. I will ping the supervisor team to understand this a bit more.

Reply:
Discussed offline, followup below.